### PR TITLE
Add script for removing old artifacts from VMware

### PR DESCRIPTION
**Pull request by jstodola**
--- Description of proposed changes ---

--- Merge policy ---

- [ ] Travis CI PASS
- [ ] `*-aws-runtest` PASS
- [ ] `*-azure-runtest` PASS
- [ ] `*-images-runtest` PASS
- [ ] `*-openstack-runtest` PASS
- [ ] `*-vmware-runtest` PASS
- [ ] For `rhel8-*` and `rhel7-*` branches commit log references an approved
  bug in Bugzilla. Do not merge if the bug doesn't have the 3 ACKs set to `+`!

--- Jenkins commands ---

- `ok to test` to accept this pull request for testing
- `test this please` for a one time test run
- `retest this please` to start a new build

### CODE REVIEW

File: tests/cleanup/remove_old_objects_vmware.sh [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+            rlLogInfo "V_USERNAME=$V_USERNAME"
+        fi
+
+        if [ -z "$V_PASSWORD" ]; then
+            rlFail "V_PASSWORD is empty!"
+        else
+            rlLogInfo "V_PASSWORD is configured"
+        fi
+
+        V_DATACENTER="${V_DATACENTER:-RH_Engineering}"
```
Comment (atodorov) [inline tests/cleanup/remove_old_objects_vmware.sh]: All up to V_FOLDER are not in use.

Comment (jstodola) [reply tests/cleanup/remove_old_objects_vmware.sh]: Right, removed from the script.

File: tests/cleanup/vmware_list_vms.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python
```
Comment (atodorov) [inline tests/cleanup/vmware_list_vms.py]: I will merge as-is but this file needs to go back into pyvmomi-community-samples (we can add it to the current branch that we use for composer_testing) until upstream decides to merge it. I will open another task for that.

**Comment by bcl** [comment]
https://travis-ci.org/weldr/lorax/builds/496938431#L4030

FYI - pylint doesn't like the way pyVmomi dynamically imports things, and the container it runs in needs to install python3-pyvmomi. I'll handle adding these when I do a build today.

**Comment by atodorov** [comment]
yikes.

@bcl we've got a ticket to move this file outside the repo and contribute it upstream. It will solve the pylint problems as well.

**Pull request merged**